app/src/main/java/com/uc/mymovies/ListAdapter.kt

```diff
@@ -12,42 +12,37 @@ import com.uc.mymovies.dto.Movie
 
 class ListAdapter(private val context: Context, var movies: ArrayList<Movie>):BaseAdapter() {
 
-    override fun getView(position: Int, convertView: View?, parent: ViewGroup?): View {
+    override fun getView(position: Int, convertView: View?, parent: ViewGroup?) : View {
         var view: View? = convertView
         val viewHolder: ViewHolder
-
-        if (view == null) {
-            viewHolder = ViewHolder()
+        view?.let {
```

Comment:
use let function to avoid if null check  
https://blog.intive-fdv.com/best-practices-to-write-a-clean-code-with-kotlin-part-ii/

Reply:
Reeeeeeeeeeeeeeally good idea here.  This is an excellent way to avoid nulls, and avoid force unwrap (!!)